libs/design/src/atoms/form/checkbox-set/checkbox-set.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input, HostBinding } from '@angular/core';
+
+@Component({
+  selector: 'daff-checkbox-set',
+  templateUrl: './checkbox-set.component.html',
+  styleUrls: ['./checkbox-set.component.scss']
+})
+export class DaffCheckboxSetComponent {
+
+  @Input() name: string;
+
+  @HostBinding('attr.role') role = 'checkbox-group';
```

Comment:
`checkbox-group` is not a valid role. It should just be `group`.

Reply:
Okay fixed in a19666774303a5986f9a4617acdc5a58ec028acf